src/types/field/index.ts

```diff
@@ -1,25 +1,45 @@
-import { IAttachmentField, IAttachmentFieldSchema } from './attachmentField'
-import { ICheckboxField, ICheckboxFieldSchema } from './checkboxField'
-import { IDateField, IDateFieldSchema } from './dateField'
-import { IDecimalField, IDecimalFieldSchema } from './decimalField'
-import { IDropdownField, IDropdownFieldSchema } from './dropdownField'
-import { IEmailField, IEmailFieldSchema } from './emailField'
-import { IHomenoField, IHomenoFieldSchema } from './homeNoField'
-import { IImageField, IImageFieldSchema } from './imageField'
-import { ILongTextField, ILongTextFieldSchema } from './longTextField'
-import { IMobileField, IMobileFieldSchema } from './mobileField'
-import { INricField, INricFieldSchema } from './nricField'
-import { INumberField, INumberFieldSchema } from './numberField'
-import { IRadioField, IRadioFieldSchema } from './radioField'
-import { IRatingField, IRatingFieldSchema } from './ratingField'
-import { ISectionField, ISectionFieldSchema } from './sectionField'
-import { IShortTextField, IShortTextFieldSchema } from './shortTextField'
-import { IStatementField, IStatementFieldSchema } from './statementField'
-import { ITableField, ITableFieldSchema } from './tableField'
-import { IUenField, IUenFieldSchema } from './uenField'
-import { IYesNoField, IYesNoFieldSchema } from './yesNoField'
+import {
+  BasicField,
+  FormField,
+  FormFieldDto,
+  MyInfoAttribute,
+} from '../../../shared/types/field'
+
+import { IAttachmentFieldSchema } from './attachmentField'
+import { ICheckboxFieldSchema } from './checkboxField'
+import { IDateFieldSchema } from './dateField'
+import { IDecimalFieldSchema } from './decimalField'
+import { IDropdownFieldSchema } from './dropdownField'
+import { IEmailFieldSchema } from './emailField'
+import { IHomenoFieldSchema } from './homeNoField'
+import { IImageFieldSchema } from './imageField'
+import { ILongTextFieldSchema } from './longTextField'
+import { IMobileFieldSchema } from './mobileField'
+import { INricFieldSchema } from './nricField'
+import { INumberFieldSchema } from './numberField'
+import { IRadioFieldSchema } from './radioField'
+import { IRatingFieldSchema } from './ratingField'
+import { ISectionFieldSchema } from './sectionField'
+import { IShortTextFieldSchema } from './shortTextField'
+import { IStatementFieldSchema } from './statementField'
+import { ITableFieldSchema } from './tableField'
+import { IUenFieldSchema } from './uenField'
+import { IYesNoFieldSchema } from './yesNoField'
+
+export * from '../../../shared/types/field/utils'
+
+export enum SPCPFieldTitle {
+  SpNric = 'SingPass Validated NRIC',
+  CpUid = 'CorpPass Validated UID',
+  CpUen = 'CorpPass Validated UEN',
+}
+
+export enum SgidFieldTitle {
+  SgidNric = 'sgID Validated NRIC',
+}
+
+export { BasicField, MyInfoAttribute, FormField }
```

Comment:
again is it really necessary to re-export? is confusing if we can import the same thing from two places